docs/releases/1.11.txt

```diff
@@ -173,6 +173,8 @@ Minor features
   :class:`~django.contrib.gis.db.models.functions.IsValid` function, and
   :lookup:`isvalid` lookup.
 
+* The OpenLayers-based form widgets are now using OpenLayers 3.
```

Comment:
are now using -> now use
(maybe combine the note with the mention about "The OpenLayers-based form widgets now use OpenLayers.js "?)